GOVERNANCE.md

```diff
@@ -0,0 +1,53 @@
+# CoreDNS Governance
+
+## Principles
+
+The CoreDNS community adheres to the following principles:
+
+- Open: CoreDNS is open source. See repository guidelines.
```

Comment:
Add a link to the guidelines

Reply:
I forgot the "below" -> "See repository guidelines, below"